tests/e2e/createWallets.spec.ts

```diff
@@ -126,10 +126,8 @@ test.describe("Create or connect wallets", () => {
     const createNewWalletButton = await getByText($document, "LNbits");
     createNewWalletButton.click();
 
-    await findByText($document, "Connect to LNbits");
-
     const lnBitsAdminKey = "d8de4f373561446aa298cae2b9424325";
-    const adminKeyField = await getByLabelText($document, "LNbits Admin Key");
+    const adminKeyField = await findByLabelText($document, "LNbits Admin Key");
```

Comment:
@Cheapshot003 couldn't make the "text with link" work.
Moved the check to "wait for label" which also achieves what we need.